hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmTransactionFactory.java

```diff
@@ -282,13 +282,12 @@ private Bytes initcodeFor(@NonNull final ContractCreateTransactionBody body) {
         if (body.hasInitcode()) {
             return body.initcode();
         } else {
-            final var maybeInitcode = fileStore.getFileLeaf(body.fileIDOrElse(FileID.DEFAULT));
-            validateTrue(maybeInitcode.isPresent(), INVALID_FILE_ID);
-            final var initcode = maybeInitcode.get();
-            validateFalse(initcode.deleted(), FILE_DELETED);
-            validateTrue(initcode.contents().length() > 0, CONTRACT_FILE_EMPTY);
+            final var initCode = fileStore.getFileLeaf(body.fileIDOrElse(FileID.DEFAULT));
```

Comment:
A nit but I believe that the original `initcode` is correct.

Reply:
I didn't change any of the code I just changed the readableFileStoreImpl and that's why I had to change in all these places

Reply:
will be fixed as part of that issue
https://github.com/hashgraph/hedera-services/issues/8207